packages/core/src/router/AbstractRouter.ts

```diff
@@ -494,6 +495,13 @@ export abstract class AbstractRouter extends Router {
     return path.replace(this._root + this._languagePartPath, '');
   }
 
+  /**
+   * Handles preManage calls for current pageManager.
+   */
+  async preManage(): Promise<unknown> {
```

Comment:
The method preManage is only for internal usage in Router. I would prefix it with ‘_’.